source/Engine.cpp

```diff
@@ -2772,9 +2772,11 @@ void Engine::FillRadar()
 	else if(!hasHostiles)
 		hadHostiles = false;
 
-	// Add projectiles that have a missile strength or homing.
-	for(Projectile &projectile : projectiles)
+	// Add projectiles that have a missile strength or blast radius.
+	for(const Projectile &projectile : projectiles)
 	{
+		if(!projectile.HasSprite())
+			continue;
 		if(projectile.MissileStrength())
```

Comment:
While we're here, I wonder if these if statements should be swapped. The currently behavior is:
1. If the projectile is a missile, display it as a white dot if it's an enemy missile (something I should worry about) and a gray dot if it's a friendly missile (something I shouldn't worry about).
2. Otherwise, if the projectile has a blast radius, display it as a larger white dot (something I should worry about regardless of who fired it).

A projectile can be a missile with blast radius, in which case it should probably be in the "something I should worry about" category. That is, unless the weapon has the "safe" tag, in which case we should probably be coloring it gray.

(And unrelated, but I wonder if blast radius projectiles should show up on the missile overlay with a special icon.)